server/views/pages/contacts/manage/listContacts.njk

```diff
@@ -28,6 +29,9 @@
     </div>
 </div>
 
+{# Test the mini profile macro #}
+{{ miniProfile(prisonerDetails, '/prisoner-image/' + prisonerDetails.prisonerNumber, digitalPrisonServicesUrl + '/prisoner/' + prisonerDetails.prisonerNumber) }}
```

Comment:
would be nice if the number of parameters could be reduced to one or two parameters may be personDetails would be enough, 

Could the url be moved into properties files and read from there, ''/prisoner-image/' and '/prisoner/'